stress/common/src/main/java/alluxio/stress/fuse/FuseIOParameters.java

```diff
@@ -28,7 +28,7 @@
   public static final String FIELD_READ_RANDOM = "mReadRandom";
 
   @Parameter(names = {"--operation"},
-      description = "the operation to perform. Options are [Read, Write]",
+      description = "the operation to perform. Options are [Read]",
```

Comment:
why remove the write operation?

Reply:
Because currently we don't support a fully-functional write throughput test

Reply:
but i can still use --operation write to write the files for the following read tests?

Reply:
Ah yes. I did this mainly because the ClientIO stress bench doesn't include write as an option in the description but it could write data for reading tests. I honestly don't know which makes more sense, include it or not..

Reply:
we can keep that for now and see how to address later.